--- origsrc/gnubiff-2.2.12/ui/Makefile.am	2010-01-10 06:41:27.000000000 -0600
+++ src/gnubiff-2.2.12/ui/Makefile.am	2010-01-31 00:53:33.973162200 -0600
@@ -4,6 +4,6 @@ desktop_DATA     = $(desktop_in_files:.d
 @INTLTOOL_DESKTOP_RULE@
 
 uidir            = $(pkgdatadir)
-dist_ui_DATA     = $(wildcard *.glade)
+dist_ui_DATA     = $(wildcard $(srcdir)/*.glade)
 
 EXTRA_DIST       = $(wildcard *.gladep) gnubiff.desktop.in
